Build Failure Due to Duplicate ibreactnative.so File in AzzappApp/react-native-skia-video and React-Android RN 0.76

What a wonderful library, video playback is much more consistent here than the RNSkia implementation, great work.

I upgraded my project to Expo 52 (52.0.37) and React Native 0.76 (0.76.7) and I am now unable to build the Android version of my app.

On build it fails with the following error:
```

> Configure project :azzapp_react-native-skia-video
react-native-skia-video: node_modules/ found at: /Users/andrewwilson/Git/azzuu-app/node_modules
react-native-skia-video: React Native version: 76

> Task :app:mergeDebugNativeLibs FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:mergeDebugNativeLibs'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.MergeNativeLibsTask$MergeNativeLibsTaskWorkAction
   > 2 files found with path 'lib/arm64-v8a/libreactnative.so' from inputs:
      - /Users/andrewwilson/Git/azzuu-app/node_modules/@azzapp/react-native-skia-video/android/build/intermediates/library_jni/debug/copyDebugJniLibsProjectOnly/jni/arm64-v8a/libreactnative.so
      - /Users/andrewwilson/.gradle/caches/8.10.2/transforms/d1ab2155b539508f564b7247055e2db0/transformed/react-android-0.76.7-debug/jni/arm64-v8a/libreactnative.so
     If you are using jniLibs and CMake IMPORTED targets, see
     https://developer.android.com/r/tools/jniLibs-vs-imported-targets
```

![Image](https://github.com/user-attachments/assets/e5da7fd5-a7c2-4e87-9d71-d4884c8b6384)

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -131,6 +131,7 @@ android {
       "**/libhermes.so",
       "**/libhermes-executor-debug.so",
       "**/libhermes_executor.so",
+      "**/libreactnative.so",
       "**/libreactnativejni.so",
       "**/libturbomodulejsijni.so",
       "**/libreact_nativemodule_core.so",

